src/components/Card.js

```diff
@@ -1,49 +1,35 @@
 import { useEffect, useState } from "react";
 import "../style/card.css";
 import { formatDate, getTimeDifference } from "../util";
+import fetchData from "../api/FetchData";
 function Card() {
   const [items, setItems] = useState(null);
 
   useEffect(() => {
-    const fetchData = async () => {
-      try {
-        const response = await fetch(
-          "https://bootcamp-api.codeit.kr/api/sample/folder"
-        );
-        const data = await response.json();
-        if (response.ok) {
-          setItems(data.folder.links);
-        }
-      } catch (e) {
-        console.error(e);
+    const fetchCardData = async () => {
+      const data = await fetchData("sample/folder");
+      if (data) {
+        setItems(data.folder.links);
       }
     };
-    fetchData();
+    fetchCardData();
   }, []);
 
   return (
-    <div className="">
-      <form className="searchLink-container">
-        <input
-          className="search-link"
-          placeholder="링크를 검색해 보세요."
-        ></input>
-      </form>
-      <div className="cardImg-grid">
-        {items &&
-          items.map((item, index) => (
-            <div className="card-container" key={index}>
-              <a href={item.url}>
-                <img className="cardImg" src={item.imageSource} alt="img"></img>
-                <p className="timeDifference">
-                  {getTimeDifference(item.createdAt)}
-                </p>
-                <p className="item-description">{item.description}</p>
-                <p>{formatDate(item.createdAt)}</p>
-              </a>
-            </div>
-          ))}
-      </div>
+    <div className="cardImg-grid">
+      {items &&
+        items.map((item, id) => (
+          <div className="card-container" key={id}>
+            <a href={item.url}>
+              <img className="cardImg" src={item.imageSource} alt="img"></img>
+              <p className="timeDifference">
+                {getTimeDifference(item.createdAt)}
+              </p>
+              <p className="item-description">{item.description}</p>
+              <p>{formatDate(item.createdAt)}</p>
+            </a>
+          </div>
+        ))}
```

Comment:
`items &&` 조건으로 배열의 존재 여부를 판단하는건 좋지만, 반환값으로 빈 배열이 내려오는 경우가 있을 수도 있어요.
그래서 일반적으로는 `items && items.length > 0`과 같이 배열의 길이가 하나 이상인 조건까지 추가해서 `map`을 돌립니다!

추가적으로 배열의 요소가 하나 이상인 경우를 판단하는 것은 자주 발생하는 조건이기 때문에 별도의 유틸 함수로 분리해서 사용하는 걸 추천드립니다!

```suggestion
      const isEmptyArray = (arr) => {
          return !arr && arr.length === 0
      }
      
      ...
      
      {isEmptyArray(items)&&
        items.map((item, id) => (
          <div className="card-container" key={id}>
            <a href={item.url}>
              <img className="cardImg" src={item.imageSource} alt="img"></img>
              <p className="timeDifference">
                {getTimeDifference(item.createdAt)}
              </p>
              <p className="item-description">{item.description}</p>
              <p>{formatDate(item.createdAt)}</p>
            </a>
          </div>
        ))}
```